bot/kodiak/test_evaluation.py

```diff
@@ -3294,7 +3294,16 @@ async def test_mergeable_include_coauthors() -> None:
     config = create_config()
     config.merge.message.include_coauthors = True
 
-    for body_style in (MergeBodyStyle.pull_request_body, MergeBodyStyle.empty):
+    for body_style, commit_message in (
+        (
+            MergeBodyStyle.pull_request_body,
+            "# some description\n\nCo-authored-by: Barry Block <73213123+b-block@users.noreply.github.com>",
```

Comment:
I think we should have more tests with some tabs and other whitespace in the commit message

Reply:
how about now that we're not trimming anything.